### PR TITLE
Issue #80: Cleanup the composer.json file.

### PR DESCRIPTION
**Pull request by drupol**
See #80

### CODE REVIEW

**Comment by drupol** [comment]
This would also fix #71 and #73

File: composer.json [hunk 1]
```diff
@@ -32,12 +36,13 @@
...
-    "openeuropa/task-runner": "^0.4",
-    "drupal/xautoload": "*"
+    "drupal/xautoload": "*",
+    "openeuropa/task-runner": "^0.4"
   },
   "autoload": {
     "psr-4": {
-      "Drupal\\Tests\\registryonsteroids\\": "./tests/src/"
+      "Drupal\\Tests\\registryonsteroids\\": "./tests/src/",
+      "Drupal\\registryonsteroids\\": "./src/"
```
Comment (donquixote) [inline composer.json]: Adding this line enables composer do the autoload for the files in src/.
This means we could now disable xautoload and tests would still pass, thus potentially hiding failures.

Is this change needed for anything?

Comment (drupol) [reply composer.json]: This is only needed for the tests, it's not used in D7.

**Comment by donquixote** [comment]
Most changes seem ok (see my other comment about the extra line in psr-4).

I would split the commit up, e.g. "add author Andreas Hennings", "remove unused requirements". I do not absolutely insist on it.

**Pull request merged**